port_ocean/config/settings.py

```diff
@@ -93,6 +93,13 @@ class IntegrationConfiguration(BaseOceanSettings, extra=Extra.allow):
     )
     max_event_processing_seconds: float = 90.0
     max_wait_seconds_before_shutdown: float = 5.0
+    multiprocessing_enabled: bool = Field(default=False)
```

Comment:
I wonder if maybe we can provide a name that will allow us more granular toggles in the future. 

runtime_mode = 'default' / 'multiprocessing' / 'parallel' 
etc...